awx/ui/src/util/getScheduleUrl.js

```diff
@@ -0,0 +1,32 @@
+export default function getScheduleUrl(job) {
+  const templateId = job.summary_fields.unified_job_template.id;
+  const scheduleId = job.summary_fields.schedule.id;
```

Comment:
hey @marshmalien, I added 2 schedules to job template, and the job details showed the schedule what triggered the job. However if I delete those schedules, I got the following error:

<img width="1790" alt="image" src="https://user-images.githubusercontent.com/9053044/155542398-bbc390b2-1466-4563-94fa-68a07986a4fe.png">

Reply:
Updated PR to handle Deleted schedules.
<img width="393" alt="Screen Shot 2022-02-28 at 3 51 07 PM" src="https://user-images.githubusercontent.com/15881645/156056916-e43658ec-ba61-4ca7-82ee-07acfba9d1be.png">